### ISSUE TITLE
Implement TextInput.submitKeyEvents for Fabric

### ISSUE THREAD

**Issue by chrisglein** [microsoft/react-native-windows#11460]
Part of https://github.com/microsoft/react-native-windows/issues/11145

Example
```jsx
      <TextInput
        accessibilityLabel="Prompt input"
        multiline={true}
        placeholder="Ask me anything"
        style={{flexGrow: 1, flexShrink: 1}}
        onChangeText={newValue => setValue(newValue)}
        submitKeyEvents={[{code: 'Enter', shiftKey: false}]}
        onSubmitEditing={submitValue}
        value={defaultText ?? value}/>
```

**Comment by YajurG** [comment]
Looks like a duplicate of #11459

**Comment by TatianaKapos** [comment]
closed via: #12746
